crates/rustc_codegen_spirv/Cargo.toml

```diff
@@ -37,7 +37,7 @@ syn = { version = "1", features = ["visit", "visit-mut"] }
 # Normal dependencies.
 bimap = "0.6"
 indexmap = "1.6.0"
-rspirv = { git = "https://github.com/gfx-rs/rspirv.git", rev = "ee1e913" }
+rspirv = { git = "https://github.com/EmbarkStudios/rspirv.git", rev = "7c72c65" }
```

Comment:
Marking this as request-change just to make sure we don't merge this (it's okay for a draft, but we shouldn't merge it)